cluster/examples/kubernetes/ceph/monitoring-roles.yaml

```diff
@@ -0,0 +1,79 @@
+#
```

Comment:
I think this file should go under the `monitoring/` directory with the rest of the prometheus stuff.

Reply:
done.